src/components/VisrNavbar.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box } from "@mui/material";
+import { Link } from "react-router-dom";
+import { Navbar, NavLink, NavLinks } from "@diamondlightsource/sci-react-ui";
+
+function VisrNavbar() {
+  return (
+    <>
+      <Navbar
+        logo="theme"
```

Comment:
If I click on the logo there's a delay and then takes me to the root. Should it just link to "/" explicitly and remove the Home link? Or is the intention that eventually it links to a higher level app?